crates/sui-open-rpc/samples/objects.json

```diff
@@ -8,22 +8,22 @@
         "fields": {
```

Comment:
What is this file used for? Is it just an example of how a transaction is serialised?